stacks-common/src/types/mod.rs

```diff
@@ -81,6 +81,12 @@ impl StacksEpochId {
     pub fn latest() -> StacksEpochId {
         StacksEpochId::Epoch24
     }
+
+    /// Returns whether or not this Epoch should perform
+    ///  Clarity value sanitization
+    pub fn value_sanitizing(&self) -> bool {
+        self >= &StacksEpochId::Epoch24
```

Comment:
For the sake of the style we agreed upon after the canonicalize types issue, should we do an explicit match for the epoch ID here instead of using a comparison operator?

Reply:
Yep, updated.